integrations/k8s-using-daemonset/k8s-with-rbac/falco-account.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
     role: security
 rules:
   - apiGroups: ["extensions",""]
-    resources: ["nodes","namespaces","pods","replicationcontrollers","services","events","configmaps"]
+    resources: ["nodes","namespaces","pods","replicationcontrollers","replicasets","services","daemonsets","deployments","events","configmaps"]
```

Comment:
`replicasets`, `daemonsets` and `deployments` are now in the `apps` API group. I think we need to extend the `apiGroups` with `apps` in order for this to work 🙂